queries.sql

```diff
@@ -8,3 +8,32 @@ select * from animal where animalname between 'Agumon' AND 'Pikachu';
 select * from animal where neutered like true;
 select * from animal where animalname not like 'Gabumon';
 select * from animal where animal_weight_kg between 10 AND 17;
+
+
+start transaction;
+update animal set species = 'unspecified';
+rollback;
+update animal set species = 'digimon' where animalname like '%mon';
+update animal set species = 'pokemon' where animalname not like '%mon';
+commit;
+select * from animal;
+start transaction;
+delete from animal;
+rollback;
+
+start transaction;
+delete from animal where animal_date_of_birth >= '2022-01-01';
+savepoint my_snapshot;
+update animal set animal_weight_kg = animal_weight_kg * -1;
+rollback to my_snapshot;
+update animal set animal_weight_kg = animal_weight_kg * -1 where animal_weight_kg < 0;
```

Comment:
@addod19 can you explain more? So I can solve this.

Like: that when I update the insert data I also share the screenshot 

Thank you

Reply:
> @addod19 can you explain more? So I can solve this.
> 
> Like: that when I update the insert data I also share the screenshot
> 
> Thank you

@Hassaanjbaig-code, yes please, because if we don't change the screenshot, the next code-reviewer will ask for it to save time. Because for example; if we have a `animal_weight_kg = 20` and `animal_weight_kg = -20` these two are not the same so definitely when we multiply by `-1` it will obviously have an effect on the query right? I hope this is clear. If not I'm here to help

Reply:
resolved! ✔️